project/plugins.sbt

```diff
@@ -0,0 +1,2 @@
+addSbtPlugin("org.scala-exercises" % "sbt-exercise" % "0.1.1", "0.13", "2.10")
```

Comment:
Fixed in this other PR
https://github.com/scala-exercises/exercises-fpinscala/pull/2


On Wed, Jul 6, 2016 at 12:19 PM Raúl Raja Martínez <notifications@github.com>
wrote:

> In project/plugins.sbt
> <https://github.com/scala-exercises/exercises-fpinscala/pull/1#discussion_r69706694>
> :
>
> > @@ -0,0 +1,2 @@
> > +addSbtPlugin("org.scala-exercises" % "sbt-exercise" % "0.1.1", "0.13", "2.10")
>
> This will also need to point to 0.2.1-SNAPSHOT
>
> —
> You are receiving this because you modified the open/close state.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/scala-exercises/exercises-fpinscala/pull/1/files/5c429fe1363c9e1cf6d454fd64ac7de40a683883#r69706694>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEp0DQKNcCiBGHt9cqeqimbgGXdo9qNBks5qS4EigaJpZM4JF7sl>
> .
>